packages/kbn-io-ts-utils/src/strict_keys_rt/index.test.ts

```diff
@@ -26,15 +24,9 @@ describe('strictKeysRt', () => {
       },
       {
         type: t.type({
-          path: t.union([
-            t.type({ serviceName: t.string }),
-            t.type({ transactionType: t.string }),
-          ]),
+          path: t.union([t.type({ serviceName: t.string }), t.type({ transactionType: t.string })]),
```

Comment:
Why the formatting changes?

Reply:
different linting rules - this was moved to `packages/`

Reply:
Okay, didn't realise we had different linting rules. nvm then